src/octoprint/plugins/action_command_notification/__init__.py

```diff
@@ -60,7 +62,20 @@ def on_event(self, event, payload):
     # ~ SettingsPlugin
 
     def get_settings_defaults(self):
-        return {"enable": True, "enable_popups": False}
+        return {"enable": True, "enable_popups": False, "regex": ""}
```

Comment:
I'd prefer we call that `filter` since it's also stored in `self._filter` and better communicates what that regex is *for*.